packages/plugin-pms-api/src/reports/utils.ts

```
@@ -0,0 +1,2638 @@
+  }
+
+  return format;
+};
+
+export const formatData = (data, filter, type) => {
+  const { dateRange, startDate, endDate, frequencyType } = filter;
+
+  const formattedData = [...data];
+
+  formattedData.forEach(item => {
+    if (item.hasOwnProperty("frequency")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)